code/API_definitions/geofencing-subscriptions.yaml

```diff
@@ -1123,16 +1034,6 @@ components:
                 status: 400
                 code: INVALID_PROTOCOL
                 message: Only HTTP is supported.
-            GENERIC_400_INVALID_CREDENTIAL:
```

Comment:
As sinkCredential should be in the request, I think these errors still make sense, as in: https://github.com/camaraproject/Commonalities/blob/main/artifacts/camara-cloudevents/event-subscription-template.yaml#L855